pkgs/applications/version-management/gitoxide/default.nix

```diff
@@ -14,14 +14,14 @@ rustPlatform.buildRustPackage rec {
   cargoSha256 = "mitUyf/z7EgjKzFy8ZER8Ceoe9tk6r0ctSYdDG87rIU=";
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ openssl ];
+  buildInputs = [ openssl ]
+    ++ stdenv.lib.optionals stdenv.isDarwin [ Security ];
```

Comment:
Just use `lib.optionals` instead of `stdenv.lib.optionals` and you don't need to replace `lib` with `stdenv` on line 1 and 23

Reply:
I think we need a rfc which says which way is right. There are many packages doing it either way right now and the diff would be at most one line shorter which does not matter tbh.